OVERVIEW.md

```diff
@@ -0,0 +1,11 @@
+# Overview
+
+## Publisher
+
+The conversion between the GraphQL JSON output and the EQ runner schema can be thought of as a pipeline.
```

Comment:
Maybe we should provide links to:

https://github.com/ONSdigital/eq-schema-validator
https://github.com/ONSdigital/eq-survey-runner

When I first joined I was unaware of these services.  Especially, https://github.com/ONSdigital/eq-schema-validator.

Reply:
Have added validator link to Overview doc, the main readme has a link to survey runner.